src/legacy/actions.ts

```diff
@@ -321,14 +321,15 @@ type DialogOptions = {
   message: string
   yesButton?: string
   noButton?: string
+  onPressYes?: () => void
```

Comment:
small changes, don't let it cascade into unrelated components

Reply:
done